src/serac/physics/utilities/functional/integral.hpp

```diff
@@ -339,7 +344,17 @@ void evaluation_kernel(const mfem::Vector& U, mfem::Vector& R, derivatives_type*
       //
       // note: make_dual(arg) promotes those arguments to dual number types
       // so that qf_output will contain values and derivatives
-      auto qf_output = qf(x_q, make_dual(arg));
+      auto qf_output = [&qf, &x_q, &arg, &data, e, q]() {
+        if constexpr (std::is_same_v<PointData, void>) {
+          // [[maybe_unused]] not supported in captures
+          (void)data;
+          (void)e;
+          (void)q;
+          return qf(x_q, make_dual(arg));
+        } else {
+          return qf(x_q, make_dual(arg), data(e, q));
+        }
+      }();
```

Comment:
let's mark this with a `TODO: refactor`, so we might add it to the list of things to revisit in an effort to clean up these kernels